storage/store.go

```diff
@@ -473,6 +475,14 @@ func (s *Store) Start(stopper *util.Stopper) error {
 	// Set the started flag (for unittests).
 	atomic.StoreInt32(&s.started, 1)
 
+	// Update the store status.
```

Comment:
You shouldn't do this in the Start() method, but only periodically after startup. You'll see why in the next comment.